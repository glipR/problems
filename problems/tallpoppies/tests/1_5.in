3 23 81
231192871 747040512 468915191 387127179 333628033 452551635 462127803 444098228 567362886 887077935 18807957 951512510 648543868 285246152 660808104 811869441 731398361 812148365 176054386 777408620 318466926 909331137 791210839
21948479 0
32878522 1 537869842
42672134 1 246063877
55227814 1 659004498
62724347 1 195139430
62797873 1 572180115
66858913 0
81708717 1 997428493
124889302 1 172961723
136535375 0
152769380 0
161081652 0
196766140 1 831451027
203456231 0
209000260 1 329539263
210128936 0
214100714 1 511324023
214657827 0
226818810 0
229232470 1 62947235
247666291 0
247838993 0
254290762 0
256048530 0
261291842 0
282270811 0
287287947 0
287789697 1 27992464
309143822 0
328477975 1 122802233
332975088 1 607949242
338230713 0
353614661 0
363408731 1 490084631
378457049 0
387503093 0
389260430 0
409058062 0
410644259 1 590363419
426938719 0
440164659 1 61272862
458153888 1 650741232
468330617 1 606806350
477040073 1 837091004
477898954 1 396839927
486713003 1 426344652
492897166 1 997861556
499896160 0
500511108 1 343193825
502616116 1 201502183
512598204 1 878126103
545986124 0
567658793 1 143201079
595445540 0
596465897 0
597869531 1 453369655
627159049 0
640368115 0
658968811 1 897383485
666812673 0
673875054 1 827757714
677142231 0
693697919 1 743165216
694847107 1 654658050
738867730 0
761855077 0
800342130 1 544425681
821969293 1 768304613
840645586 0
843771992 1 694903040
850638998 0
864638735 1 684276614
896398595 0
921890710 1 747933096
931883436 1 642112178
948248026 1 277143010
956450098 0
957063299 0
959395717 0
966369241 0
983376620 0
