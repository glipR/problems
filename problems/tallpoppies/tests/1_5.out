1017357947
1109907876
1296684060
1270456505
1295114397
1401579532
1407795428
1396275823
1427496942
1474397234
1404325083
1411416154
1335508476
1277196582
1315727624
1323991644
1379983101
1458790367
1500078259
1522498326
1546295775
1501409323
1545641112
1566062309
1740841087
1869151243
1962391006
1907445166
1900285104
1928070734
1869459900
1829720510
1919141673
1894589209
2041106816
1957887779
2089054223
2244600119
2229620539
2218518642
2175387291
2151944359
